students/jinsu/users/models.py

```diff
@@ -0,0 +1,16 @@
+from django.db import models
+
+# Create your models here.
+class User(models.Model):
+    name     = models.CharField(max_length=20)
+    email    = models.CharField(max_length=100, unique=True)
+    password = models.CharField(max_length=32)
+    contact  = models.CharField(max_length=20)
+    age      = models.PositiveIntegerField()
+
+    class Meta:
+        db_table = 'users'
```

Comment:
```suggestion
class User(models.Model):
    name     = models.CharField(max_length=45)
    email    = models.CharField(max_length=200, unique=True)
    password = models.CharField(max_length=200)
    contact  = models.CharField(max_length=45)
    age      = models.PositiveIntegerField()

    class Meta:
        db_table = 'users'
```
진수님 변수명, 컨벤션, Field 지정 너무 좋습니다! max_length는 넉넉하게 잡아주는 게 좋으니 수정해주세요!
contact 정보에는 보통 핸드폰 번호, 이메일, 개인 사이트 주소 등이 올 수 있으니 어떤 데이터가 담길지 유추하기 어렵습니다. 구체적으로 수정해주세요!

Reply:
제안 사항 반영하여 max_length를 늘리고, contact 변수명을 phone_number로 수정하였습니다.